donate/templates/payment/card_upsell_master.html

```diff
@@ -37,19 +37,19 @@ <h1 class="heading heading--primary heading--bottom-margin">{% trans "Thank you
 
                         <div class="form__group form__group--additional-info">
                             <span class="form-item__input-additional-info form-item__input-additional-info--left" data-donation-currency>{% get_localized_currency_symbol currency_info.code %}</span>
-                            <span class="form-item__input-additional-info">{% trans "/ Monthly" %}</span>
+                            <span class="form-item__input-additional-info">{% trans "Monthly" %}</span>
                             {% render_form_field form.amount %}
                         </div>
 
-                        <p><small>{% trans "I authorize Mozilla Foundation to charge my card in the amount shown above and to continue doing so each month until I cancel." %}</small></p>
+                        <p class="donate-form__authorization">{% trans "I authorize Mozilla Foundation to automatically charge my card every month in the amount indicated above, starting next month on the same date as today, and continuing each month afterwards until I cancel. I understand that I must cancel at least 5 days before the next scheduled donation by email to donate@mozilla.org, and that I may only request a refund within 15 days from the date of a donation." %}</p>
```

Comment:
Let’s use a placeable for the email here too. Is it intended we’re not making it a mailto: link? I assume so, because it’s not an immediate action, but just making sure